pitch.md

```diff
@@ -67,3 +86,5 @@ _Why is this a good project? What are some challenges you expect to face? How
 might you overcome them? What are some warning signs that the project has gotten
 off track, and how will you get the project back on track if needed?_
 
+I think this is a good project because it solves a problem that is relevant to my life but also to the life of almost every student at the 5Cs and colleges everywhere. I expect to face significant problems with building the parser and designing the syntax and I bet that the constraint solver itself will be pretty hard as well. I am thinking a lot about how I will actually weight the constraints. Most of these challenges are challenges that simply require me to work hard and learn in order to overcome them. I think that I will know that the project has gotten off track if I start feeling like i _dont want to work on it_. When I'm doing CS homework I almost always am excited to do it and like to dive in and spend hours and hours on it but I also have gotten to times in assignments that are going badly where I just abandon them in the back of my mind. If this happens I will leverage my advisor at Pomona (prof Chen) as well as Prof. Ben to help figure out what is going wrong and how I can get back on track mentally, as well as with the coding.
```

Comment:
That's an interesting perspective on how to tell you're off-track.

I would add that another possible way to get off-track is if you ended up diving really deep into adjacent problems, like maybe "What really makes a good group?" and such, which I'm sure are endless, or in working with creating (but not designing) the GUI.